src/components/notification/channel/lark.ts

```diff
@@ -118,7 +118,15 @@ export const sendLark = async (
                 },
                 {
                   tag: 'text',
-                  value: `${message.meta.averageResponseTime} ms in the last 24 hours`,
+                  value: `${message.meta.maxResponseTime} ms in the run time`,
```

Comment:
instead of `in the run time`, can we get the value of duration since the last status notification? if not, better to say `ms since the last status report.`

Reply:
that's outdated code. i have changed it to in the last X hours. with minimum is 1 hour